nautobot/extras/models/jobs.py

```diff
@@ -864,6 +877,7 @@ def enqueue_job(
             "nautobot_job_job_model_id": job_model.id,
             "nautobot_job_profile": profile,
             "nautobot_job_user_id": user.id,
+            "nautobot_job_ignore_singleton_lock": ignore_singleton_lock,
```

Comment:
Many of the other `nautobot_job_*` celery kwargs get handled in `nautobot/core/celery/backends.py` and `nautobot/core/celery/schedulers.py`. Do we need similar code added for this parameter?

Reply:
I'm afraid I don't quite understand the consequences here.

- Why would a scheduled job behave differently in this regard?
- What is the purpose of `backends.py`?

Reply:
`NautobotDatabaseBackend` handles the automatic update of JobResult records. I think if we move `ignore_singleton_lock` into `celery_kwargs` it should be fine there since we're already passing around `celery_kwargs` in that code.